src/pathrewriter.ts

```diff
@@ -24,15 +24,18 @@ import constants from './constants';
 
 const pathPosix = path.posix;
 
-class PathResolver {
-  constructor(public abspath?: boolean) {
+class PathRewriter {
```

Comment:
This class has no real state outside `abspath`, which is only used in `rewriteRelPath`. It should probably just be a module of functions.